src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -41,14 +42,16 @@ trait PostgresClient {
   /*
    * Run a single SELECT query and wrap the results with the provided function.
    */
-  def select[T](sql: String)
-    (f: Row => T): Future[Seq[T]]
+  def select[T](sql: String)(f: Row => T): Future[Seq[T]] =
+    selectToStream(sql)(f).flatMap(_.toSeq)
+  def selectToStream[T](sql: String)(f: Row => T): Future[AsyncStream[T]]
```

Comment:
Nit: Can this be called `selectStream`?

Reply:
Sure thing... Though should we also rename `prepareAndQueryToStream` to `prepareAndQueryStream`? I find that somewhat questionable, but I don't really have a strong opinion.

Reply:
Hmm, I agree with you about that... I'm not totally sure TBH. I don't want to bikeshed the API too much, as long as it works people probably don't care all that much 😀

Let me give it some thought